src/main/java/ch/njol/skript/expressions/ExprAmount.java

```diff
@@ -41,14 +45,20 @@
  */
 @Name("Amount")
 @Description({"The amount of something.",
-		"Please note that <code>amount of %items%</code> will not return the number of items, but the number of stacks, e.g. 1 for a stack of 64 torches. To get the amount of items in a stack, see the <a href='#ExprItemAmount'>item amount</a> expression."})
+		"Please note that <code>amount of %items%</code> will not return the number of items, but the number of stacks, e.g. 1 for a stack of 64 torches. To get the amount of items in a stack, see the <a href='#ExprItemAmount'>item amount</a> expression.",
+		"Also please note that getting a list's recursive size can cause lag if the list is large."})
```

Comment:
Documentation for recursive size is insufficient. If even I don't know what exactly it does without reading implementation, something is wrong.